### PR TITLE
Ruby 2.4.3

### PR DESCRIPTION
**Pull request by alexeymorozov**

### CODE REVIEW

**Comment by alexeymorozov** [comment]
Tests require approval to run, but they pass [here](https://github.com/alexeymorozov/ruby-packer/actions/runs/843237264).

@ronaldtse @CAMOBAP Please take a look.

**Comment by ronaldtse** [comment]
@alexeymorozov you have access here now. 

Please help remove the Travis build, and fee free to merge when all the builds pass (the ones that supposed to pass, that is). Thanks!

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+  macos-test:
+    needs: macos-packed-mn
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.ignore-errors }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ macos-10.15 ]
+        ignore-errors: [ false ]
+        test-flavor:
```
Comment (CAMOBAP) [inline .github/workflows/build.yml:217]: I propose to reduce this list just to iso

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+    needs: linux-packed-mn
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.ignore-errors }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-16.04, ubuntu-18.04, ubuntu-20.04 ]
+        ignore-errors: [ false ]
+        test-flavor:
+          - iso
```
Comment (CAMOBAP) [inline .github/workflows/build.yml:116]: I propose to reduce this list just to iso

**Review by CAMOBAP — APPROVED** [review]
LGTM, just reduce test matrix to speedup CI

File: .github/workflows/build.yml (hunk 1 above)
Comment (ronaldtse) [reply .github/workflows/build.yml:217]: Agree!

File: .github/workflows/build.yml (hunk 2 above)
Comment (ronaldtse) [reply .github/workflows/build.yml:116]: Agree.

**Pull request merged**